code/game/objects/items/weapons/material/twohanded.dm

```diff
@@ -157,3 +157,18 @@
 	fragile = 1	//It's a haphazard thing of glass, wire, and steel
 	reach = 2 // Spears are long.
 	attackspeed = 14
+
+//This is mostly for centaurs.
+/obj/item/weapon/material/twohanded/spear/lance
+	name = "lance"
+	desc = "End him rightly"

```

Comment:
Gonna need a pommel for that one.

Reply:
Like, a jousting lance? Is that not the correct term for that?

Reply:
Sorry, I was mostly joking -- referring to https://knowyourmeme.com/memes/end-him-rightly, popularized by a guy named Skallagrim

But yeah a pommel is a piece from the bottom of a sword's grip, not quite a lance